main.cc

```diff
@@ -1756,13 +1756,7 @@ static main_func_type lookup_main_func(std::string_view name) {
         {"perf-simple-query", perf::scylla_simple_query_main},
         {"perf-sstable", perf::scylla_sstable_main},
     };
```

Comment:
At this point this can be just moved to `main()`, no reason to keep in a separate method.

Reply:
sure, will create another commit to move it into `main()`.